### ISSUE TITLE
Different or no focus tag on some components (2.4.7 Synlig fokus (Nivå AA))

### ISSUE THREAD

**Issue by olavflar** [Altinn/app-frontend-react#2838]
### Description of the bug

After doing some testing for updating the UU status for Altinn 3, I discovered that there was some different focus tags. And some components where missing 

2.4.7 Synlig fokus (Nivå AA)

Innhold som bryter kravet

På grunn av en feil i løsningen har noen komponenter mistet en av markører som indikerer at elementet har tastaturfokus. Til vanlig skal det være en mørk og en gul ramme rundt hele komponenten. I de tilfellene feilen slår inn er kun den gule rammen synlig. Den har ikke god nok kontrast alene. Dette vil føre til at tastaturbrukere ikke vil se hvor de har fokus. Feilen skal fikses. 

### Steps To Reproduce

Go to https://ttd.apps.tt02.altinn.no/ttd/frontend-test#/
And tab around to find the components missing or having another focus ring 

### Additional Information

<img width="360" alt="Image" src="https://github.com/user-attachments/assets/95b2e658-176e-4a3d-9018-6c3bb44f9e80" />
<img width="434" alt="Image" src="https://github.com/user-attachments/assets/14b35a7a-32bb-4d93-9f06-61389494ea00" />
<img width="453" alt="Image" src="https://github.com/user-attachments/assets/32dd0d83-9354-4cb9-8bde-c035de18e892" />
<img width="955" alt="Image" src="https://github.com/user-attachments/assets/8ee424d6-4d52-40e3-9c4d-c22290f7affa" />
<img width="952" alt="Image" src="https://github.com/user-attachments/assets/85aacdc1-1199-4247-8d73-021c2c974c7e" />
<img width="1049" alt="Image" src="https://github.com/user-attachments/assets/8d60c63b-949c-4577-aaea-2572ad747ce8" />
<img width="1051" alt="Image" src="https://github.com/user-attachments/assets/adf0207d-3c72-4103-9b7e-c52ab2d7ccc2" />
<img width="748" alt="Image" src="https://github.com/user-attachments/assets/04a698c3-10fb-44d0-8032-adc0befb05e1" />
<img width="604" alt="Image" src="https://github.com/user-attachments/assets/8daa8cee-fb4f-456c-bf67-83c8523a9d14" />
